### ISSUE TITLE
Dev Tools

### ISSUE THREAD

**Issue by Krutsch** [developit/ama#7]
Hi Jason,

if you would spin up a new web app, that uses typescript, would you make use of eslint? For my opinion, eslint is kind of redundant in this case.
And what about styles? Do you use postcss, stylelint, scss? If you don't have a strong opinion about this, than what would the Preact Team think about it?

Cheers

**Comment by developit** [comment]
I think [wmr](https://github.com/preactjs/wmr) might answer some of these bits, but FWIW I very rarely write pure TypeScript, so ESLint lets me apply rules and analysis across JS and TS alike. I don't generally use ESLint for stylistic stuff much anymore, I use it more for things like picking up breakages when using hooks. 

I've never used StyleLint, but I do generally use PostCSS in projects. I like the SCSS syntax, but I don't use enough sass features to justify it. PostCSS gives me nesting, modules, composition, etc. As an example, in WMR we shipped PostCSS by default, and we only support SCSS if you install one of the Sass implementations. 

I guess my personal philosophy here is basically to reevaluate the layers of abstraction I rely on every once in a while to see if they're still needed. If I can get rid of a piece of tech I have relied on, that often helps simplify things so I don't end up debugging them later on.
